core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/WorkflowSubmissionActor.scala

```diff
@@ -420,6 +421,7 @@ trait WorkflowSubmission extends FutureSupport with LazyLogging with MethodWiths
                                                          RawlsUserEmail(submissionRec.submitterEmail)
       )
       petUserInfo <- googleServicesDAO.getUserInfoUsingJson(petSAJson)
+      samUserInfo <- samDAO.getUserStatus(RawlsRequestContext(petUserInfo))
```

Comment:
This is a new api call out to sam on workflow submissions right? I think it'll be a drop in the ocean on requests to sam but figured it was worth calling out.

Reply:
Yeah, this is a new call, but one that Bard would have had to make if Rawls was sending to Bard. Just the price of tying events to a user.